docs/python/tutorial-django.md

```diff
@@ -443,17 +450,20 @@ In production, you also need to set `DEBUG=False` in `settings.py`, which necess
     }
     ```
 
-1. In `templates/hello/hello_there.html`, add the following lines after the `<title>` element. The `{% load static %}` tag is a custom Django template tag set, which allows you to use `{% static %}` to refer to a file like the stylesheet.
+    You should see the syntax highlighting that VS Code provides for CSS files, including a color preview.
+
+    ![Django tutorial: CSS Syntax Highlighting](images/django-tutorial/css-syntax-highlighting.png)
+
+1. In `templates/hello/hello_there.html`, add `{% load static %}` above `<!DOCTYPE html>` and then add the following line after the `<title>` element. The `{% load static %}` tag is a custom Django template tag set, which allows you to use `{% static %}` to refer to a file like the stylesheet.
 
     ```html
-    {% load static %}
     <link rel="stylesheet" type="text/css" href="{% static 'hello/site.css' %}" />
     ```
 
 1. Also in `templates/hello/hello_there.html`, replace the contents `<body>` element with the following markup that uses the `message` style instead of a `<strong>` tag:
 
     ```html
-    <span class="message">Hello, there \{{ name }}!</span> It's \{{ date | date:'l, d F, Y' }} at \{{ date | time:'H:i:s' }}.
+    <span class="message">Hello, there \{{ name }}!</span> It's \{{ date|date:'l, d F, Y' }} at \{{ date|time:'H:i:s' }}.
     ```
```

Comment:
Not only are the spaces between variables, "|", and the following filters non-standard practice, VS Code insisted on formatting this by placing the closing `}}` on a new line after I pasted it into the template, which of course breaks the tag. I could only get this template snippet to work by removing the spaces and then VS Code left it alone, all on one line. I documented this behavior above, but it's unclear what the guidance should be for new users. "If VS Code keeps breaking your code on save, turn off all formatting or try another IDE that doesn't do that"?